### PR TITLE
[Docs] update time window functions

### LINKED ISSUE #2029: Document `tumbleXYZ` functions.
Create documentation for the `tumbleXYZ` function. See #1833 for details on how to complete this issue.

- [ ] tumbleEnd
- [ ] tumbleStart

Child issue of #1833.

### PR DESCRIPTION
**Pull request by Blargian**
Closes [#2029](https://github.com/ClickHouse/clickhouse-docs/issues/2029) as part of the [functions project](https://github.com/ClickHouse/clickhouse-docs/issues/1833) to document missing / insufficiently documented functions.

- makes updates to formatting of time window functions page
- adds missing examples for `start`/`end` variants of `tumble` and `hop`.

### Changelog category (leave one):
- Documentation (changelog entry is not required)

### CODE REVIEW

**Comment by Blargian** [comment]
@rschu1ze tagging for review

File: docs/en/sql-reference/functions/time-window-functions.md [hunk 1]
```diff
@@ -6,111 +6,237 @@ sidebar_label: Time Window
...
+tumbleStart(time_attr, interval [, timezone]);
 ```
 
 **Arguments**
 
-- `time_attr` - Date and time. [DateTime](../data-types/datetime.md) data type.
-- `hop_interval` - Hop interval in [Interval](../data-types/special-data-types/interval.md) data type. Should be a positive number.
-- `window_interval` - Window interval in [Interval](../data-types/special-data-types/interval.md) data type. Should be a positive number.
-- `timezone` — [Timezone name](../../operations/server-configuration-parameters/settings.md#server_configuration_parameters-timezone) (optional). 
+- `time_attr` — Date and time. [DateTime](../data-types/datetime.md).
```
Comment (rschu1ze) [inline docs/en/sql-reference/functions/time-window-functions.md:59]: Should we document `bounds_tuple` (l. 53)?

**Comment by robot-clickhouse-ci-2** [comment]
*This is an automated comment for commit eb12d4e37e63212e831ccb92fc3c9818da7ba866 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/66102/eb12d4e37e63212e831ccb92fc3c9818da7ba866/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Style check</td><td>Runs a set of checks to keep the code style clean. If some of tests failed, see the related log from the report</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/66102/eb12d4e37e63212e831ccb92fc3c9818da7ba866/style_check.html">✅ success</a></td></tr>
<tbody>
</table>
</details>

File: docs/en/sql-reference/functions/time-window-functions.md (hunk 1 above)
Comment (Blargian) [reply docs/en/sql-reference/functions/time-window-functions.md:59]: Having `tumbleStart(bounds_tuple)` in addition to `tumbleStart(time_attr, interval [, timezone])` in the syntax feels unnecessary to me. Maybe we just keep the latter and add a note saying that it's also possible to pass a tuple of those parameters?

Comment (rschu1ze) [reply docs/en/sql-reference/functions/time-window-functions.md:59]: Yes, sounds like a plan!

**Pull request merged**